### PR TITLE
Add glossary of functions used in each lesson

### PR DESCRIPTION
**Pull request by juanfung**
Updated `reference.md` a la R Ecology to be a glossary of functions used in each episode #267

### CODE REVIEW

File: reference.md [hunk 1]
```diff
@@ -4,4 +4,93 @@ layout: reference
...
+  * `pivot_wider()` # reshape a data frame by a key-value pair across multiple columns
+  * `pivot_longer()` # reshape a data frame by collapsing into a key-value pair
+  * `replace_na()`  # Replace NAs with specified values
+  * `n_distinct()` # get a count of unique values
+  * `write_csv()` # save to a csv formatted file
+
+### Lesson 4 -- Data Visualization with ggplot2
+
+  * `read_csv()` # load a csv formatted file into R memory
+  * `ggplot2(data= , aes(x= , y= )) + geom_point( ) + facet_wrap () + theme_bw() + theme() `
```
Comment (kelseygonzalez) [inline reference.md:81]: Should we add an explanation that this is skeleton code?

Comment (juanfung) [reply reference.md:81]: Yes, I can do that. Should it be on the same line or in a footnote?

**Review by kelseygonzalez — COMMENTED** [review]
It looks great except the one small comment I left. Possible to change and then approve, though not necessary.

File: reference.md (hunk 1 above)
Comment (kelseygonzalez) [reply reference.md:81]: Hm, the rest are in the same line right after the code, so perhaps the same as the other line?

Comment (juanfung) [reply reference.md:81]: Gotcha, didn't realize there was no description. Thanks!

**Comment by juanfung** [comment]
@kelseygonzalez Thanks! Change made. H/T to R Ecology.

**Review by atheobold — APPROVED** [review]
This looks good to me!

**Pull request merged**